ext/date/date_core.c

```diff
@@ -6732,13 +6732,13 @@ cmp_dd(VALUE self, VALUE other)
  *
  *     d <=> DateTime.new(2022, 7, 26) # => 1
  *     d <=> DateTime.new(2022, 7, 27) # => 0
- *     d <=> DateTime.new(2022, 7, 29) # => -1
+ *     d <=> DateTime.new(2022, 7, 28) # => -1
  *
- * - A numeric (compares <tt>self.ajd</tt> to +other+):
+ * - A numeric (compares <tt>self.jd</tt> to +other+):
```

Comment:
Why was this changed? It's compares `self.ajd`.

Reply:
Copy/edit mistake.